frontend/src/component/project/Project/ProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -336,8 +372,13 @@ export const ProjectFeatureToggles = ({
                                 name: env,
                                 enabled:
                                     featureEnvironments?.find(
-                                        feature => feature?.name === env
+                                        fe => fe?.name === env
```

Comment:
It's nice that we changed `feature` to `fe` to better represent what we're dealing with here, since it's an environment from the feature, not the feature itself. I would still prefer something more explicit, like `environment` or `featureEnvironment` instead of `fe`.

Reply:
Well, yes, but it gets too long. I chose `fe` because we're using that acronym elsewhere and also because it's close to the collection `featureEnvironments` so you can quickly look it up, but having `feature` was misleading because you'd trust it to be a feature without looking at the definition. Still, I agree `featureEnvironment` could be a good variable name

Reply:
[bdf3e4a](https://github.com/Unleash/unleash/pull/2948/commits/bdf3e4a126d8f889369ae62060c10deb11c41694) as gratitude for all your help :hugs:

Reply:
Ah, thanks! 😁 I think I got the habit of being more extensive and explicit with my naming because of PR comments like these, especially from @FredrikOseberg - Otherwise I always opted for short variable names in these cases. 

But I think it's a good habit to have and it doesn't represent that much more effort for potentially more readability and clarity.